### ISSUE TITLE
Customize UI: Tint Color

### ISSUE THREAD

**Issue by lucianoenrico** [ideashower/ShareKit#289]
Hi,
 I wanted to customize the Sharekit top bar's tint color. I used these settings in SHKConfig.h with no luck.

```
#define SHKBarTintColorRed          -1 // Value between 0-255, set all to -1 for default
#define SHKBarTintColorGreen        -1 // Value between 0-255, set all to -1 for default
#define SHKBarTintColorBlue         -1 // Value between 0-255, set all to -1 for default
```

I searched the whole Sharekit source code but I did not found where these #define are used. It seems that they're only defined but never used. It's true?

if so where I can customize the top bar tint color?

thank you.

**Comment by natefanaro** [comment]
I had the same problem. See this comment in issue #202 for a fix https://github.com/ideashower/ShareKit/issues/202#issuecomment-1549976

**Comment by lucianoenrico** [comment]
the fix in issue #202 solves the problem.
Thank you.
